vuu-ui/packages/vuu-shell/src/layout-management/layoutTypes.ts

```diff
@@ -6,10 +6,15 @@ export type LayoutMetadata = {
   screenshot: string;
   user: string;
   date: string;
-  id: string;
 };
 
+export type PersistedLayoutMetadata = {
+  id: string,
+  metadata: LayoutMetadata
+}
+
 export type Layout = {
+  id: string,
   json: LayoutJSON;
   metadata: LayoutMetadata;
 };
```

Comment:
this type here could be {json: LayoutJSON} & PersistedLayoutMetadata

Reply:
I've taken a slightly different approach to these types now, which I think relates better to how entities would be stored in a database. I've also restructured the local storage to align with this.